### PR TITLE
[ready for review] Adding figure and alt-text to "The Environmental Impact of Digital Research" page

### PR DESCRIPTION
**Pull request by kallewesterling**
Adding figure and alt-text to "The Environmental Impact of Digital Research" page. Partially addresses #3331.

### CODE REVIEW

File: book/website/ethical-research/activism/activism-env-impact.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 (er-activism-env-impact)=
 # The Environmental Impact of Digital Research
 
+```{figure} ../../figures/environmental-impact.*
+---
+height: 400px
+name: environmental-impact
+alt: Cartoon-like sketch depicting the potential environmental impact of digital research. On the left, a person sits at a desk with a laptop, with a chatbot and "COMPUTING" text above, symbolizing digital communication. In the center, "EMISSIONS" emerge from a factory, representing pollution. On the right, a figure throws computers into the ocean, labeled "WASTE," indicating electronic disposal issues.
```
Comment (Arielle-Bennett) [inline book/website/ethical-research/activism/activism-env-impact.md:8]:
```suggestion
alt: Cartoon-like sketch depicting the potential environmental impact of digital research. On the left, a person sits at a desk with a laptop, with a chatbot and "COMPUTING" text above, symbolizing digital communication. In the center, "EMISSIONS" emerge from a factory, representing pollution. On the right, a figure throws computers into the ocean, labeled "WASTE", indicating electronic disposal issues.

```

I would also edit this to add the colour scheme to the alt-text - fixed a minor typo as well 👍🏻

**Comment by kallewesterling** [comment]
@Arielle-Bennett if you don't mind reviewing the revisions I made, that'd be great :)

**Review by llewelld — COMMENTED** [review]
Your alt-text description really captures both the imagery and the intention of the image in my opinion; I think it works really well. The caption looks good to me, although I admit I'm not an expert here, so it may be worth checking with someone who is.

This is a really nice addition to the page.

File: book/website/ethical-research/activism/activism-env-impact.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
 (er-activism-env-impact)=
 # The Environmental Impact of Digital Research
 
+```{figure} ../../figures/environmental-impact.*
+---
+height: 400px
```
[the remark this answers is not in the document]
Comment (Arielle-Bennett) [reply book/website/ethical-research/activism/activism-env-impact.md:6]: @kallewesterling can you commit this and then we'll see what it looks like? After that I'm happy to approve and get this merged, the alt-text looks great 😄

[the remark this answers is not in the document]
Comment (kallewesterling) [reply book/website/ethical-research/activism/activism-env-impact.md]: Just changed with https://github.com/the-turing-way/the-turing-way/pull/3376/commits/6954f9496babfead29a50238e036dc36746926d9. 

A note though, @Arielle-Bennett - should we add something about this to [the section in the Community Handbook/Style Guide](https://the-turing-way.netlify.app/community-handbook/style/style-figures#myst-syntax-to-add-a-figure-to-a-turing-way-chapter)?

[the remark this answers is not in the document]
Comment (kallewesterling) [reply book/website/ethical-research/activism/activism-env-impact.md]: FYI @Arielle-Bennett added this issue: https://github.com/the-turing-way/the-turing-way/issues/3415

**Pull request merged**